lib/music_theory/note.rb

```diff
@@ -1,17 +1,19 @@
 # coding: utf-8
 require 'music_theory/output'
 require 'music_theory/pitch'
+require 'music_theory/envelope'
 
 module MusicTheory
   class Note
     include MusicTheory::Output
-    attr_accessor :frequency, :pitch, :duration, :output_file_name, :distort
+    attr_accessor :frequency, :pitch, :duration, :output_file_name, :distort, :envelope
 
     def initialize(options = {})
       @frequency        = options[:pitch] ? options[:pitch].frequency
-                          : options[:frequency].to_f || 440     # pitch ex) Pitch.new(:Ds, 3)
+                          : options[:frequency].to_f || 440     # pitch ex: Pitch.new(:Ds, 3)
```

Comment:
well if `options[:frequency]` is `nil` you'll get an error for trying to call `#to_f` on `nil`